wiki/Multiplayer/en.md

```diff
@@ -1,451 +1,419 @@
 
-[osu!academy][osu!academy wikilink] covered this page in [Episode 6 (6:52)][osu!academy Ep6] along with [Online Users Panel][Chat_Console#Extended_Chat_Console wikilink].
+Multiplayer or _Multi_ was implemented on the 3rd of June 2008 (3.6.08) under [osu! Public Release b335][Release b335], and is a game mode in which up to sixteen players can compete against each other on a single map decided by the host.
+
+[osu!academy][osu!academy wikilink] covered this page in [Episode 6 (6:52)][osu!academy Ep6] along with [Online Users Panel][Chat_Console#Extended_Chat Console wikilink].
 
 ## How to enter
+
 ### Minimum requirements
+
+![One of the ways to transfer the beatmap without leaving the room][Beatmaps_peppy image]
+
+_One of the ways to transfer the beatmap without leaving the room_
+
 - Decent online connection.
-- osu!account logged into Bancho (should be done automatically)
-- Have a fair amount of beatmaps in your Songs folder (unless you prefer to download as you go)
-- Knowing [how to transfer beatmaps without leaving the room.][Beatmaps2 wikilink] [Picture example here.][Beatmaps_peppy image]
-- Logical thinking (knowing when to ask about something you don't understand)
+- _osu!account_ logged into _Bancho_ (should be done automatically during ingame login)
+- Have a fair amount of beatmaps in the `Songs` folder (except for personal preference to download on the spot)
+- Knowing [how to transfer beatmaps without leaving the room.][Beatmaps2 wikilink].
+  - Opening the file directly usually works.
 
 ### Ways to enter
+
 From Main menu,
-- Press "**Play**" then "**Multi**"; or
-- Press "**P**" then "**M**". (from Keyboard)
 
-You will be directed to the "**lobby**", the core of the Multi-play.
+- Press `Play` then `Multi`; or
+- Press `P` then `M`. (from Keyboard)
+
+If the connection request with _Bancho_ was successful, _osu!_ will be directed to the **lobby**, the core of the Multi-play.
 
 ### Notes
-- Keep in mind that downloading times can range from a few seconds to 15mins depending on your internet connection.
-- Losing connection to Bancho will send you back to Main Menu until connection is recovered to Bancho. This will force you to be considered "Quit" when playing a match.
-- "Quit" when playing a match.
-- Requires [osu!supporter][osu!supporter wikilink] in cuttingedge build.
+
+Keep in mind that downloading times can range from a few seconds to 15 minutes onwards depending on internet connection.
+
+Losing connection to _Bancho_ will direct _osu!_ back to Main Menu until connection to _Bancho_ has been recovered.
+When playing a match, a "Quit" will be given on the playerbox.
+
+If _not_ in _Stable_ build, the logged _osu!account_ must has an active _[osu!supporter][osu!supporter wikilink]_ to enter Multi.
```

Comment:
_OBJECTION!!!_
`If you are using the _cuttingedge_ build, the logged-in _osu!account_ must have an active _[osu!supporter][osu!supporter wikilink]_ tag to enter.`